cert/lib/cert/options.rb

```diff
@@ -48,6 +48,15 @@ def self.available_options
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_TEAM_NAME"] = value.to_s
                                      end),
+        FastlaneCore::ConfigItem.new(key: :filename,
+                                     short_option: "-q",
+                                     env_name: "CERT_FILE_NAME",
+                                     optional: true,
+                                     description: "The filename of certificate to store (must include .cer)",
```

Comment:
Instead of the `verify_block`

Reply:
Yes we could 👍 But for consistency I have kept arguments written in `sigh` options : [sigh#L102-L109](https://github.com/fastlane/fastlane/blob/master/sigh/lib/sigh/options.rb#L102-L109).

Reply:
But if we auto append file extension, we should keep the `verify_block` for checking filename doesn't contain a file extension, right ?

Reply:
@jhanzo I like your thought on consistency so if anything I would be down for also changing `sigh` in the future to auto append the extension if needs it (since its easier on end users for us to auto append what we are looking for)

In this case, I agree with @KrauseFx the we should move the `must include '.cer'` from the description and also remove the `verify_block` 👍 This will make this addition a lot more forgiving for new users 🙃